### ISSUE TITLE
Docker error response when trying to create a new project

### ISSUE THREAD

**Issue by martinaelena** [docksal/docksal#771]
**Description**

when I run fin project create I get an error:

Cloning repository...
docker: Error response from daemon: invalid mount config for type "bind": bind source path does not exist.
See 'docker run --help'.

I've tried most things. restarting vm. even setting up a new vm. still get the same error.

**Steps to reproduce the issue:**
mac os 10.12.6 ---- virtual box
folder:  /Users/teneramirez/docksal (has all my projects)
1. fin project create: 
2. enter your choice: 1. 
3. do you wish to proceed: y

**Describe the results you received:**
Cloning repository...
docker: Error response from daemon: invalid mount config for type "bind": bind source path does not exist.
See 'docker run --help'.

**Describe the results you expected:**

expected it to create a drupal project successfully

**Output of `fin config`:**

<details>
  <summary>fin config output</summary>
**this first config is from a working project**

```
---------------------
COMPOSE_PROJECT_NAME_SAFE: bflaw
COMPOSE_FILE:
/Users/teneramirez/.docksal/stacks/volumes-bind.yml
/Users/teneramirez/.docksal/stacks/stack-default.yml
/Users/teneramirez/docksal/bflaw/.docksal/docksal.yml
ENV_FILE:
/Users/teneramirez/docksal/bflaw/.docksal/docksal.env

PROJECT_ROOT: /Users/teneramirez/docksal/bflaw
DOCROOT: docroot
VIRTUAL_HOST: bflaw.docksal
VIRTUAL_HOST_ALIASES: *.bflaw.docksal
IP: 192.168.64.100
MYSQL: 192.168.64.100:32768

Docker Compose configuration
---------------------
services:
  cli:
    dns:
    - 192.168.64.100
    - 8.8.8.8
    environment:
      DOCROOT: docroot
      HOST_GID: '20'
      HOST_UID: '501'
      SECRET_ACAPI_EMAIL: null
      SECRET_ACAPI_KEY: null
      SECRET_BLACKFIRE_CLIENT_ID: null
      SECRET_BLACKFIRE_CLIENT_TOKEN: null
      SECRET_PLATFORMSH_CLI_TOKEN: null
      SECRET_SSH_PRIVATE_KEY: null
      SECRET_TERMINUS_TOKEN: null
      VIRTUAL_HOST: bflaw.docksal
      XDEBUG_CONFIG: remote_connect_back=0 remote_host=192.168.64.1
      XDEBUG_ENABLED: '0'
    hostname: cli
    image: docksal/cli:2.4-php7.1
    volumes:
    - docksal_ssh_agent:/.ssh-agent:ro
    - cli_home:/home/docker:rw
    - project_root:/var/www:rw,nocopy
  db:
    dns:
    - 192.168.64.100
    - 8.8.8.8
    environment:
      MYSQL_ALLOW_EMPTY_PASSWORD: null
      MYSQL_DATABASE: default
      MYSQL_INITDB_SKIP_TZINFO: null
      MYSQL_ONETIME_PASSWORD: null
      MYSQL_PASSWORD: user
      MYSQL_RANDOM_ROOT_PASSWORD: null
      MYSQL_ROOT_PASSWORD: root
      MYSQL_USER: user
    hostname: db
    image: docksal/db:1.2-mysql-5.6
    ports:
    - 0:3306/tcp
    volumes:
    - db_data:/var/lib/mysql:rw
    - project_root:/var/www:ro,nocopy
  web:
    depends_on:
      cli:
        condition: service_started
    dns:
    - 192.168.64.100
    - 8.8.8.8
    environment:
      APACHE_BASIC_AUTH_PASS: null
      APACHE_BASIC_AUTH_USER: null
      APACHE_DOCUMENTROOT: /var/www/docroot
    hostname: web
    image: docksal/web:2.1-apache2.4
    labels:
      io.docksal.cert-name: none
      io.docksal.project-root: /Users/teneramirez/docksal/bflaw
      io.docksal.virtual-host: bflaw.docksal,*.bflaw.docksal,bflaw.docksal.*
    volumes:
    - project_root:/var/www:ro,nocopy
version: '2.1'
volumes:
  cli_home: {}
  db_data: {}
  docksal_ssh_agent:
    external: true
    name: docksal_ssh_agent
  project_root:
    driver: local
    driver_opts:
      device: /Users/teneramirez/docksal/bflaw
      o: bind
      type: none

---------------------
```

**this config is from the non working folder**

```
martina:kctg teneramirez$ fin config
---------------------
COMPOSE_PROJECT_NAME_SAFE: kctg
COMPOSE_FILE:
/Users/teneramirez/.docksal/stacks/volumes-bind.yml
/Users/teneramirez/.docksal/stacks/stack-default.yml
/Users/teneramirez/docksal/kctg/.docksal/docksal.yml
ENV_FILE:
/Users/teneramirez/docksal/kctg/.docksal/docksal.env

PROJECT_ROOT: /Users/teneramirez/docksal/kctg
DOCROOT: docroot
VIRTUAL_HOST: kctg.docksal
VIRTUAL_HOST_ALIASES: *.kctg.docksal
IP: 192.168.64.100
MYSQL:

Docker Compose configuration
---------------------
services:
  cli:
    dns:
    - 192.168.64.100
    - 8.8.8.8
    environment:
      DOCROOT: docroot
      HOST_GID: '20'
      HOST_UID: '501'
      SECRET_ACAPI_EMAIL: null
      SECRET_ACAPI_KEY: null
      SECRET_BLACKFIRE_CLIENT_ID: null
      SECRET_BLACKFIRE_CLIENT_TOKEN: null
      SECRET_PLATFORMSH_CLI_TOKEN: null
      SECRET_SSH_PRIVATE_KEY: null
      SECRET_TERMINUS_TOKEN: null
      VIRTUAL_HOST: kctg.docksal
      XDEBUG_CONFIG: remote_connect_back=0 remote_host=192.168.64.1
      XDEBUG_ENABLED: '0'
    hostname: cli
    image: docksal/cli:2.4-php7.1
    volumes:
    - docksal_ssh_agent:/.ssh-agent:ro
    - cli_home:/home/docker:rw
    - project_root:/var/www:rw,nocopy
  db:
    dns:
    - 192.168.64.100
    - 8.8.8.8
    environment:
      MYSQL_ALLOW_EMPTY_PASSWORD: null
      MYSQL_DATABASE: default
      MYSQL_INITDB_SKIP_TZINFO: null
      MYSQL_ONETIME_PASSWORD: null
      MYSQL_PASSWORD: user
      MYSQL_RANDOM_ROOT_PASSWORD: null
      MYSQL_ROOT_PASSWORD: root
      MYSQL_USER: user
    hostname: db
    image: docksal/db:1.2-mysql-5.6
    ports:
    - 3306/tcp
    volumes:
    - db_data:/var/lib/mysql:rw
    - project_root:/var/www:ro,nocopy
  web:
    depends_on:
      cli:
        condition: service_started
    dns:
    - 192.168.64.100
    - 8.8.8.8
    environment:
      APACHE_BASIC_AUTH_PASS: null
      APACHE_BASIC_AUTH_USER: null
      APACHE_DOCUMENTROOT: /var/www/docroot
    hostname: web
    image: docksal/web:2.1-apache2.4
    labels:
      io.docksal.cert-name: none
      io.docksal.project-root: /Users/teneramirez/docksal/kctg
      io.docksal.virtual-host: kctg.docksal,*.kctg.docksal,kctg.docksal.*
    volumes:
    - project_root:/var/www:ro,nocopy
version: '2.1'
volumes:
  cli_home: {}
  db_data: {}
  docksal_ssh_agent:
    external: true
    name: docksal_ssh_agent
  project_root:
    driver: local
    driver_opts:
      device: /Users/teneramirez/docksal/kctg
      o: bind
      type: none

---------------------
```
</details>

<br>

**Output of `fin sysinfo`:**

<details>
  <summary>fin sysinfo output</summary>

```
███  OS
Darwin Mac OS X 10.12.6
Darwin martina.home 16.7.0 Darwin Kernel Version 16.7.0: Fri Apr 27 17:59:46 PDT 2018; root:xnu-3789.73.13~1/RELEASE_X86_64 x86_64

███  ENVIRONMENT
MODE : VirtualBox VM
DOCKER_HOST : tcp://192.168.64.100:2376
DOCKSAL_NFS_PATH : /Users/teneramirez/docksal
NFS EXPORTS:
# VAGRANT-BEGIN: 501 0ddadf4b-41e1-4f72-8e94-adf0936d86a3
"/Users/teneramirez/Documents/mmgy" 192.168.56.101 -alldirs -mapall=501:20
# VAGRANT-END: 501 0ddadf4b-41e1-4f72-8e94-adf0936d86a3
# VAGRANT-BEGIN: 501 d0c25c58-7314-46f1-a36a-d02090a0cce4
#"/Users/teneramirez/Documents/sites" 192.168.56.101 -alldirs -mapall=501:20
# VAGRANT-END: 501 d0c25c58-7314-46f1-a36a-d02090a0cce4
# VAGRANT-BEGIN: 501 01e64291-b437-4b69-91d9-98778c332ebd
#"/Users/teneramirez/Documents/mesites" 192.168.56.101 -alldirs -mapall=501:20
# VAGRANT-END: 501 01e64291-b437-4b69-91d9-98778c332ebd
# <ds-nfs docksal
/Users/teneramirez/docksal 127.0.0.1 192.168.64.100 -alldirs -maproot=0:0
# ds-nfs>

███  FIN
fin version: 1.69.1

███  DOCKER COMPOSE
EXPECTED VERSION: 1.21.1
docker-compose version 1.21.1, build 5a3f1a3
docker-py version: 3.3.0
CPython version: 3.6.4
OpenSSL version: OpenSSL 1.0.2o  27 Mar 2018

███  DOCKER
EXPECTED VERSION: 18.03.1-ce

Client:
 Version:      18.03.1-ce
 API version:  1.37
 Go version:   go1.9.5
 Git commit:   9ee9f40
 Built:        Thu Apr 26 07:13:02 2018
 OS/Arch:      darwin/amd64
 Experimental: false
 Orchestrator: swarm

Server:
 Engine:
  Version:      18.03.1-ce
  API version:  1.37 (minimum version 1.12)
  Go version:   go1.9.5
  Git commit:   9ee9f40
  Built:        Thu Apr 26 07:23:03 2018
  OS/Arch:      linux/amd64
  Experimental: false

███  DOCKER MACHINE
EXPECTED VERSION: 0.14.0
docker-machine version 0.14.0, build 89b8332

NAME      ACTIVE   DRIVER       STATE     URL                         SWARM   DOCKER        ERRORS
docksal   *        virtualbox   Running   tcp://192.168.64.100:2376           v18.03.1-ce
me        -        virtualbox   Stopped                                       Unknown

███  DOCKSAL: PROJECTS
project             STATUS              virtual host                                    project root
bflaw               Up 28 hours         bflaw.docksal,*.bflaw.docksal,bflaw.docksal.*   /Users/teneramirez/docksal/bflaw
msb                 Up 23 hours         msb.docksal,*.msb.docksal,msb.docksal.*         /Users/teneramirez/docksal/msb
ops                 Up 25 hours         ops.docksal,*.ops.docksal,ops.docksal.*         /Users/teneramirez/docksal/ops

███  DOCKSAL: VIRTUAL HOSTS
http://*.bflaw.docksal
http://*.msb.docksal
http://*.ops.docksal
http://bflaw.docksal.*
http://bflaw.docksal
http://msb.docksal.*
http://msb.docksal
http://ops.docksal.*
http://ops.docksal

███  DOCKER: RUNNING CONTAINERS
CONTAINER ID        IMAGE                       COMMAND                  CREATED             STATUS                  PORTS                                                    NAMES
8008df2399a8        docksal/web:2.1-apache2.4   "httpd-foreground"       28 hours ago        Up 28 hours             80/tcp, 443/tcp                                          bflaw_web_1
dbd01ba43a87        docksal/cli:2.4-php7.1      "/opt/startup.sh sup…"   28 hours ago        Up 28 hours (healthy)   22/tcp, 3000/tcp, 9000/tcp                               bflaw_cli_1
0f1328f0e1b0        docksal/db:1.2-mysql-5.6    "/entrypoint.sh mysq…"   28 hours ago        Up 28 hours             0.0.0.0:32768->3306/tcp                                  bflaw_db_1
4a50cdae0d5e        docksal/web:2.1-apache2.4   "httpd-foreground"       29 hours ago        Up 23 hours             80/tcp, 443/tcp                                          msb_web_1
17f3555cbbd6        docksal/cli:2.4-php7.1      "/opt/startup.sh sup…"   29 hours ago        Up 23 hours (healthy)   22/tcp, 3000/tcp, 9000/tcp                               msb_cli_1
9d2f552acd81        docksal/db:1.2-mysql-5.6    "/entrypoint.sh mysq…"   29 hours ago        Up 23 hours             0.0.0.0:32770->3306/tcp                                  msb_db_1
d1371acb366c        docksal/ssh-agent:1.0       "/run.sh ssh-agent"      29 hours ago        Up 28 hours                                                                      docksal-ssh-agent
88c061e74ef8        docksal/dns:1.0             "/opt/entrypoint.sh …"   29 hours ago        Up 28 hours             192.168.64.100:53->53/udp                                docksal-dns
84c991193e49        docksal/vhost-proxy:1.3     "docker-entrypoint.s…"   29 hours ago        Up 28 hours             192.168.64.100:80->80/tcp, 192.168.64.100:443->443/tcp   docksal-vhost-proxy
0e8c191a743f        docksal/db:1.2-mysql-5.6    "/entrypoint.sh mysq…"   2 days ago          Up 25 hours             0.0.0.0:32769->3306/tcp                                  ops_db_1
a253e02b20fd        docksal/web:2.1-apache2.4   "httpd-foreground"       2 days ago          Up 25 hours             80/tcp, 443/tcp                                          ops_web_1
b0a922bec14d        docksal/cli:2.4-php7.1      "/opt/startup.sh sup…"   2 days ago          Up 25 hours (healthy)   22/tcp, 3000/tcp, 9000/tcp                               ops_cli_1

███  DOCKER: NETWORKS
NETWORK ID          NAME                DRIVER              SCOPE
656c8071c3f8        _default            bridge              local
98e0404723f7        bflaw_default       bridge              local
9c8185eb4af4        bridge              bridge              local
cd466becbbc9        host                host                local
4bc156715948        kc_default          bridge              local
e34ade02d572        kctg_default        bridge              local
9cc6c7f6f920        maloney_default     bridge              local
807417f11e8f        me_default          bridge              local
322b6aa03b5e        msb_default         bridge              local
bfbe0ecb1d1b        none                null                local
50e9c1660a06        ops_default         bridge              local
f57d2226a2bc        tester_default      bridge              local

███  VIRTUALBOX
EXPECTED VERSION: 5.2.2
5.2.2r119230

███  DOCKSAL MOUNTS
192.168.64.1:/Users/teneramirez/docksal on /Users/teneramirez/docksal type nfs (rw,noatime,nodiratime,vers=3,rsize=65536,wsize=65536,namlen=255,acregmin=1,acregmax=1,acdirmin=1,acdirmax=1,hard,nocto,nolock,proto=tcp,port=2049,timeo=70,retrans=3,sec=sys,local_lock=all,addr=192.168.64.1)
192.168.64.1:/Users/teneramirez/docksal/bflaw on /mnt/sda1/var/lib/docker/volumes/bflaw_project_root/_data type nfs (rw,noatime,nodiratime,vers=3,rsize=65536,wsize=65536,namlen=255,acregmin=1,acregmax=1,acdirmin=1,acdirmax=1,hard,nocto,nolock,proto=tcp,port=2049,timeo=70,retrans=3,sec=sys,local_lock=all,addr=192.168.64.1)
192.168.64.1:/Users/teneramirez/docksal/ops on /mnt/sda1/var/lib/docker/volumes/ops_project_root/_data type nfs (rw,noatime,nodiratime,vers=3,rsize=65536,wsize=65536,namlen=255,acregmin=1,acregmax=1,acdirmin=1,acdirmax=1,hard,nocto,nolock,proto=tcp,port=2049,timeo=70,retrans=3,sec=sys,local_lock=all,addr=192.168.64.1)
192.168.64.1:/Users/teneramirez/docksal/msb on /mnt/sda1/var/lib/docker/volumes/msb_project_root/_data type nfs (rw,noatime,nodiratime,vers=3,rsize=65536,wsize=65536,namlen=255,acregmin=1,acregmax=1,acdirmin=1,acdirmax=1,hard,nocto,nolock,proto=tcp,port=2049,timeo=70,retrans=3,sec=sys,local_lock=all,addr=192.168.64.1)

███  HDD Usage
Filesystem                Size      Used Available Use% Mounted on
/dev/sda1                46.1G      4.9G     38.8G  11% /mnt/sda1
```

</details>

**Comment by achekulaev** [comment]
From chat:

```
achekulaev> try executing `fin run-cli ls` and tell if you see the same error.
martinaelena> Yes, same error if I run the above command.
```

Something is wrong with `run-cli` command on this host in general. But I am not sure what.

**Comment by justinlevi** [comment]
I'm seeing the same error as well and with `fin run-cli ls`

**Comment by martinaelena** [comment]
I have not been able to resolve my issue. Once I get my projects done, I plan on reinstallation of items.

**Comment by achekulaev** [comment]
I guess there is a windows-related issue in `run-cli` function. I will investigate. For the moment you could use workaround by checking out respective repo manually and then running `fin init`.

**Comment by karlkedrovsky** [comment]
I'm getting this on OSX so it might not be Windows specific. I did a "fin update" earlier and then saw this error when I ran "fin project create". I then went into an existing project that's using Docksal ran "fin up" (which worked just fine) then ran "fin run-cli ls" which produced the error in this issue.

**Output of `fin sysinfo`:**

<details><summary>fin sysinfo output</summary>
███  OS
Darwin Mac OS X 10.12.6
Darwin kkedrovsky2-mac 16.7.0 Darwin Kernel Version 16.7.0: Thu Jun 21 20:07:39 PDT 2018; root:xnu-3789.73.14~1/RELEASE_X86_64 x86_64

███  ENVIRONMENT
MODE : VirtualBox VM
DOCKER_HOST : tcp://192.168.64.100:2376
DOCKSAL_NFS_PATH : /Users/kkedrovsky/projects-docksal
NFS EXPORTS:
/Users/kkedrovsky/projects -mapall=470876382 -network 10.1.0.0 -mask 255.255.255.0
# <ds-nfs docksal
/Users/kkedrovsky/projects-docksal 127.0.0.1 192.168.64.100 -alldirs -maproot=0:0
# ds-nfs>

███  FIN
fin version: 1.69.1

███  DOCKER COMPOSE
EXPECTED VERSION: 1.21.1
docker-compose version 1.21.1, build 5a3f1a3
docker-py version: 3.3.0
CPython version: 3.6.4
OpenSSL version: OpenSSL 1.0.2o  27 Mar 2018

███  DOCKER
EXPECTED VERSION: 18.03.1-ce

Client:
 Version:      18.03.1-ce
 API version:  1.37
 Go version:   go1.9.5
 Git commit:   9ee9f40
 Built:        Thu Apr 26 07:13:02 2018
 OS/Arch:      darwin/amd64
 Experimental: false
 Orchestrator: swarm

Server:
 Engine:
  Version:      18.03.1-ce
  API version:  1.37 (minimum version 1.12)
  Go version:   go1.9.5
  Git commit:   9ee9f40
  Built:        Thu Apr 26 07:23:03 2018
  OS/Arch:      linux/amd64
  Experimental: false

███  DOCKER MACHINE
EXPECTED VERSION: 0.14.0
docker-machine version 0.14.0, build 89b8332

NAME      ACTIVE   DRIVER       STATE     URL                         SWARM   DOCKER        ERRORS
docksal   *        virtualbox   Running   tcp://192.168.64.100:2376           v18.03.1-ce

███  DOCKSAL: PROJECTS
project             STATUS              virtual host                                                               project root
drupalcorn2018      Up 5 minutes        drupalcorn2018.docksal,*.drupalcorn2018.docksal,drupalcorn2018.docksal.*   /Users/kkedrovsky/projects-docksal/drupalcorn2018

███  DOCKSAL: VIRTUAL HOSTS
http://*.drupalcorn2018.docksal
http://drupalcorn2018.docksal.*
http://drupalcorn2018.docksal

███  DOCKER: RUNNING CONTAINERS
CONTAINER ID        IMAGE                       COMMAND                  CREATED             STATUS                   PORTS                                                    NAMES
0cba4d088f7f        docksal/web:2.1-apache2.4   "httpd-foreground"       5 minutes ago       Up 5 minutes             80/tcp, 443/tcp                                          drupalcorn2018_web_1
be90446d58ce        docksal/cli:2.4-php7.1      "/opt/startup.sh sup…"   5 minutes ago       Up 5 minutes (healthy)   22/tcp, 3000/tcp, 9000/tcp                               drupalcorn2018_cli_1
5c4f3c5c8c1e        docksal/ssh-agent:1.0       "/run.sh ssh-agent"      3 hours ago         Up 2 hours                                                                        docksal-ssh-agent
166d874b7d78        docksal/dns:1.0             "/opt/entrypoint.sh …"   3 hours ago         Up 2 hours               192.168.64.100:53->53/udp                                docksal-dns
66a305e46c45        docksal/vhost-proxy:1.3     "docker-entrypoint.s…"   3 hours ago         Up 2 hours               192.168.64.100:80->80/tcp, 192.168.64.100:443->443/tcp   docksal-vhost-proxy
e571cd516064        docksal/db:1.2-mysql-5.6    "/entrypoint.sh mysq…"   2 weeks ago         Up 5 minutes             0.0.0.0:32768->3306/tcp                                  drupalcorn2018_db_1

███  DOCKER: NETWORKS
NETWORK ID          NAME                       DRIVER              SCOPE
3e29f19c7056        _default                   bridge              local
5b6dee868ebd        bricks_default             bridge              local
5a001524dfa4        bridge                     bridge              local
add174b104fa        drupalcorn2018_default     bridge              local
4bf9eba19fe5        hopehouse_default          bridge              local
7c6558793caa        host                       host                local
fa5fcccbf689        none                       null                local
7048b38e595d        related-rentals_default    bridge              local
d1ccd816dad0        rockhurst_default          bridge              local
1191c10eac28        trabon-solutions_default   bridge              local

███  VIRTUALBOX
EXPECTED VERSION: 5.2.2
5.2.18r124319

███  DOCKSAL MOUNTS
192.168.64.1:/Users/kkedrovsky/projects-docksal on /Users/kkedrovsky/projects-docksal type nfs (rw,noatime,nodiratime,vers=3,rsize=65536,wsize=65536,namlen=255,acregmin=1,acregmax=1,acdirmin=1,acdirmax=1,hard,nocto,nolock,proto=tcp,port=2049,timeo=70,retrans=3,sec=sys,local_lock=all,addr=192.168.64.1)
192.168.64.1:/Users/kkedrovsky/projects-docksal/drupalcorn2018 on /mnt/sda1/var/lib/docker/volumes/drupalcorn2018_project_root/_data type nfs (rw,noatime,nodiratime,vers=3,rsize=65536,wsize=65536,namlen=255,acregmin=1,acregmax=1,acdirmin=1,acdirmax=1,hard,nocto,nolock,proto=tcp,port=2049,timeo=70,retrans=3,sec=sys,local_lock=all,addr=192.168.64.1)

███  HDD Usage
Filesystem                Size      Used Available Use% Mounted on
/dev/sda1                46.1G      8.9G     34.8G  20% /mnt/sda1
</details>

**Comment by chluther** [comment]
Same issue.  When `run fin project` create or `fin run-cli` I get error:
`docker: Error response from daemon: invalid mount config for type "bind": bind source path does not exist.`

On `$ fin sysinfo output`
███  OS
Darwin Mac OS X 10.13.6
Darwin ITs-MacBook-Pro 17.7.0 Darwin Kernel Version 17.7.0: Wed Oct 10 23:06:14 PDT 2018; root:xnu-4570.71.13~1/RELEASE_X86_64 x86_64

███  ENVIRONMENT
MODE : VirtualBox VM
DOCKER_HOST : tcp://192.168.64.100:2376
DOCKSAL_NFS_PATH : /Volumes/easystore/Projects
NFS EXPORTS:

# <ds-nfs docksal
/Volumes/easystore/Projects 127.0.0.1 192.168.64.100 -alldirs -maproot=0:0
# ds-nfs>

███  FIN
fin version: 1.69.1

███  DOCKER COMPOSE
EXPECTED VERSION: 1.21.1
docker-compose version 1.21.1, build 5a3f1a3
docker-py version: 3.3.0
CPython version: 3.6.4
OpenSSL version: OpenSSL 1.0.2o  27 Mar 2018

███  DOCKER
EXPECTED VERSION: 18.03.1-ce

Client:
 Version:      18.03.1-ce
 API version:  1.37
 Go version:   go1.9.5
 Git commit:   9ee9f40
 Built:        Thu Apr 26 07:13:02 2018
 OS/Arch:      darwin/amd64
 Experimental: false
 Orchestrator: swarm

Server:
 Engine:
  Version:      18.03.1-ce
  API version:  1.37 (minimum version 1.12)
  Go version:   go1.9.5
  Git commit:   9ee9f40
  Built:        Thu Apr 26 07:23:03 2018
  OS/Arch:      linux/amd64
  Experimental: false

███  DOCKER MACHINE
EXPECTED VERSION: 0.14.0
docker-machine version 0.14.0, build 89b8332

NAME      ACTIVE   DRIVER       STATE     URL                         SWARM   DOCKER        ERRORS
docksal   *        virtualbox   Running   tcp://192.168.64.100:2376           v18.03.1-ce   

███  DOCKSAL: PROJECTS
project             STATUS              virtual host        project root

███  DOCKSAL: VIRTUAL HOSTS
No virtual hosts found

███  DOCKER: RUNNING CONTAINERS
CONTAINER ID        IMAGE                     COMMAND                  CREATED             STATUS              PORTS                                                    NAMES
00b3fb8f5a76        docksal/ssh-agent:1.0     "/run.sh ssh-agent"      23 seconds ago      Up 22 seconds                                                                docksal-ssh-agent
03957e9b6550        docksal/dns:1.0           "/opt/entrypoint.sh …"   30 seconds ago      Up 29 seconds       192.168.64.100:53->53/udp                                docksal-dns
deba7b729ecc        docksal/vhost-proxy:1.3   "docker-entrypoint.s…"   31 seconds ago      Up 30 seconds       192.168.64.100:80->80/tcp, 192.168.64.100:443->443/tcp   docksal-vhost-proxy

███  DOCKER: NETWORKS
NETWORK ID          NAME                           DRIVER              SCOPE
76d4602b3ff3        _default                       bridge              local
fda0db26c202        bridge                         bridge              local
df9421095178        host                           host                local
cd5f7c039b74        none                           null                local
47fb3cb3b5ef        pfsamplesaustrgrvau2_default   bridge              local
b8416ea6f8ef        test_default                   bridge              local

███  VIRTUALBOX
EXPECTED VERSION: 5.2.2
5.2.20r125813

███  DOCKSAL MOUNTS
192.168.64.1:/Volumes/easystore/Projects on /Volumes/easystore/Projects type nfs (rw,noatime,nodiratime,vers=3,rsize=65536,wsize=65536,namlen=255,acregmin=1,acregmax=1,acdirmin=1,acdirmax=1,hard,nocto,nolock,proto=tcp,port=2049,timeo=70,retrans=3,sec=sys,local_lock=all,addr=192.168.64.1)

███  HDD Usage
Filesystem                Size      Used Available Use% Mounted on
/dev/sda1                46.1G      1.8G     41.9G   4% /mnt/sda1

**Comment by lmakarov** [comment]
Fixed via #829
